modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -91,31 +91,114 @@ The following snippet configures an Ethernet interface that uses a dynamic IP ad
+====
+
+The following examples show situations that require configuring a network interface to store DNS values:
+
+[IMPORTANT]
+====
+You cannot use `br-ex` bridge, an OVNKubernetes-managed Open vSwitch bridge, as the interface when configuring DNS resolvers.
+====
+
+* Configure a static DNS for a network interface with an automatic IP configuration. Note that for this configuration, you must set the `auto-dns` parameter to `false`, so that the Kubernetes NMState Operator can store custom DNS settings for the network interface.
++
 [source,yaml]
 ----
+dns-resolver:
+  config:
+    search:
+    - example.com
+    - example.org
+    server:
+    - 2001:db8:f::1
+    - 192.0.2.251
+interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+    ipv4:
+      enabled: true
+      dhcp: true
+      auto-dns: false
+    ipv6:
+      enabled: true
+      dhcp: true
+      autoconf: true
+      auto-dns: false
 # ...
-    interfaces: <1>
-       ...
-       ipv4:
-         ...
-         auto-dns: false
-         ...
-    dns-resolver:
-      config:
-        search:
-        - example.com
-        - example.org
-        server:
-        - 8.8.8.8
+----
+
+* Configure a static DNS for a network interface with a static IP configuration. Note that for this configuration, you must set the `dhcp` parameter to `false` and the `autoconf` parameter to `false`.
++
+[source,yaml]
+----
+dns-resolver:
+  config:
+# ...
+    server:
+    - 2001:4860:4860::8844
+    - 192.0.2.251
+interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+    ipv4:
+      enabled: true
+      dhcp: false
+      address:
+      - ip: 192.0.2.251
+        prefix-length: 24
+    ipv6:
+      enabled: true
+      dhcp: false
+      autoconf: false
+      address:
+      - ip: 2001:db8:1::1
+        prefix-length: 64
+routes:
+  config:
+  - destination: 0.0.0.0/0
+    next-hop-address: 192.0.2.1
+    next-hop-interface: eth1
+  - destination: ::/0
+    next-hop-address: 2001:db8:1::3
+    next-hop-interface: eth1
 # ...
 ----
-<1> You must configure an interface with `auto-dns: false` or you must use static IP configuration on an interface in order for Kubernetes NMState to store custom DNS settings.
 
+* Configure a static DNS name server to append to Dynamic Host Configuration Protocol (DHCP) and IPv6 Stateless Address AutoConfiguration (SLAAC) servers:
++
+[source,yaml]
+----
+dns-resolver:
+  config:
+# ...
+    server:
+    - 192.0.2.251
+interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+    ipv4:
+      enabled: true
+      dhcp: true
+      auto-dns: true
+    ipv6:
+      enabled: true
+      dhcp: true
+      autoconf: true
+      auto-dns: true
+# ...
+----
++
 [IMPORTANT]
 ====
-You cannot use `br-ex`, an OVNKubernetes-managed Open vSwitch bridge, as the interface when configuring DNS resolvers.
+Do not define `server` and `auto-dns` parameters for this server configuration.
```

Comment:
I'm confused by this note. We're defining both of those parameters in the configuration above.

Reply:
I recently deleted an example so this note stemmed from that. My apologies. Removed the note.